src/config/laravel-analytics.php

```diff
@@ -11,17 +11,12 @@
      * Path to the client secret json file. Take a look at the README of this package
      * to learn how to get this file.
      */
-    'service_account_credentials_json' => storage_path('app/laravel-google-analytics/service-account-credentials.json'),
+    'service_account_credentials_json' => env('GOOGLE_APPLICATION_CREDENTIALS', storage_path('app/laravel-google-analytics/service-account-credentials.json')),
```

Comment:
I think it's indeed unnecessary in the config file. 

If people disagree with that decision they can easily put an env variable there themselves.